extensions/guacamole-auth-duo/src/main/resources/translations/it.json

```diff
@@ -0,0 +1,11 @@
+{
+
+    "DATA_SOURCE_DUO" : {
+     },
```

Comment:
There's no reason to include a completely empty section.

Reply:
Deleted empty section, I hope to have understand what you mean